cli/azd/test/functional/cli_test.go

```diff
@@ -44,6 +44,10 @@ import (
 	"go.opentelemetry.io/otel/attribute"
 )
 
+const (
+	testSubscriptionId = "2cd617ea-1866-46b1-90e3-fffb087ebf9b"
```

Comment:
This works great for us for now, but I think long term we will need to figure out a better story here, since this test is now only runnable by folks that have access to this subscription (they could always change this value but then they have to worry about not including this hunk as part of their changes).

We chatted a bit about this, and I think we both figured that long term we should try to remove this functional test in favor of something closer to a unit test style test where we wouldn't be required to have a valid subscription ID to run the test?  The reason this was a functional test was because it was written at a point where we had no good way to write unit level tests for this logic.

Do you mind filing a bug about cleaning this up?  Feel free to just assign it to me if you'd like, I'd love to get to it during MQ.

Reply:
Created issue #1019